lib/sidekiq/error_tag.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Sidekiq::ErrorTag
+  def call(_worker, job, _queue)
+    Raven.tags_context(job: job['class'].underscore)
```

Comment:
I think I'd be ok with the class name as is; there's some (admittedly small) mental overhead to underscoring the job you're querying for. Also does the class come in demodulized? (if not `::` becomes `/` via underscore)

Reply:
sounds good, i won't underscore the class name. the full class name is sent, for example `EVSS::ClaimsJob`